content/en/docs/setup/install/multicluster/index.md

```diff
@@ -0,0 +1,281 @@
+            --from-file=ca-cert.pem \
+            --from-file=ca-key.pem \
+            --from-file=root-cert.pem \
+            --from-file=cert-chain.pem \
+            --dry-run -o yaml| kubectl apply -f -
+    $ popd
+    $ kubectl create clusterrole istio-reader-istio-system --verb=get,list,watch \
+        --resource=nodes,pods,endpoints,services,replicasets.apps,replicationcontrollers --dry-run -o yaml \
+        | kubectl apply -f -
+     
+    $ kubectl create serviceaccount istio-reader-service-account -n istio-system  --dry-run -o yaml \
+        | kubectl apply -f -
+     
+    $ kubectl create clusterrolebinding istio-reader-istio-system --clusterrole=istio-reader-istio-system \
+         --serviceaccount=istio-system:istio-reader-service-account  --dry-run -o yaml \
+        | kubectl apply -f -
+    EOF
+    $ chmod +x prepca.sh
+    $ istioctl x multicluster generate -f mesh-topology.yaml --from base.yaml > istio-$(kubectl config current-context).yaml
+    $ istioctl manifest apply -f istio-$(kubectl config current-context).yaml
+    $ for CONTEXT in $(kubectl config get-context -o name); do
+        $ istioctl --context ${CONTEXT} x multicluster generate --from base.yaml -f mesh-topology.yaml > istio-${CONTEXT}.yaml
+        $ istioctl --context ${CONTEXT} manifest apply -f istio-${CONTEXT}.yaml
+    $ done
+    {{< /text >}}
+
+1. Now create an offline root CA by running the following:
+
+    {{< text bash >}}
+    $ mkdir mesh-certs && pushd mesh-certs
+    $ curl -O @samples/certs/Makefile@
+    $ export ROOTCA_ORG=mesh.example.com
+    $ export CITADEL_ORG=mesh.example.com
+    $ make root-ca
+    $ popd
+    {{< /text >}}
+
+    TBD: What should we recommend for thr ROOTCA_ORG and CITADEL_ORG values?
+    TBD: What advice can we give about keeping this secret?
+
+1. Finally, create a topology file. This will be used to hold the set of clusters you'll be adding to the mesh:
+
+    {{< text bash >}}
+    $ cat > mesh-topology.yaml << EOF
+    mesh_id: MyMeshID 
+    clusters:
+    EOF
+    {{< /text >}}
+    
+    You should set the mesh_id field to a reasonably unique and descriptive name for your mesh. This will help you identity your meshes in the
+    future.
+
+## Setting up the first cluster
+
+1. Select the first cluster's context:
+
+    {{< text bash >}}
+    $ kubectl config use-context context-1
+    {{< /text >}}
+    
+1. Register the first cluster in the topology file. `mesh-topology.yaml` should now contain:
+
+    {{< text yaml >}}
+    mesh_id: MyMeshID 
+    clusters:
+      context-1:
+        network: network-1
+        serviceAccountReader: istio-reader-service-account        
+    {{< /text >}}
+
+1. Prepare the cluster by running the `prepcluster.sh` script created above:
+
+    {{< text bash >}}
+    $ ./prepcluster.sh
+    {{< /text >}}
+
+1. Apply the topology information:
+
+    {{< text bash >}}
+    $ istioctl x multicluster apply -f mesh-topology.yaml
```

Comment:
Here would be a good place to add the generate command:

```
istioctl x multicluster generate -f mesh-topology.yaml --from base.yaml \
     > istio-$(kubectl config current-context).yaml
```

Reply:
And I see now that this relies on Istio being installed on line 103. Are you presuming three existing Istio clusters that are being joined, or new clusters being configured and joined at the same time?